pkg/pillar/scripts/device-steps.sh

```diff
@@ -191,13 +191,13 @@ fi
 
 P3=$(zboot partdev P3)
 P3_FS_TYPE=$(blkid "$P3"| awk '{print $3}' | sed 's/TYPE=//' | sed 's/"//g')
-if [ -c $TPM_DEVICE_PATH ] && ! [ -f $CONFIGDIR/disable-tpm ] && [ "$P3_FS_TYPE" = "ext4" ]; then
+if [ -c $TPM_DEVICE_PATH ] && [ "$P3_FS_TYPE" = "ext4" ]; then
```

Comment:
Are you sure we will never need to disable use of the TPM as part of brining up new hardware platforms?

Reply:
We at least need disable-tpm to be able to run with LISP right now.

Reply:
Added back "disable-tpm" check.